test/networkd-test.py

```diff
@@ -594,6 +594,162 @@ def test_dropin(self):
         self.assertIn('nameserver 192.168.42.1\n', contents)
         self.assertIn('nameserver 127.0.0.1\n', contents)
 
+
+class UnmanagedClientTest(ClientTestBase, unittest.TestCase):
```

Comment:
I've added another commit that makes a separate utility class that I've added to.  (Sorry for the delay in that.)  More functionality could probably be migrated to it in a later pass.